db_revisions/versions/4e8ae26c1a22_create_filing_task_state_table.py

```diff
@@ -21,8 +21,10 @@
 def upgrade() -> None:
     op.create_table(
         "filing_task_state",
-        sa.Column("filing", sa.INTEGER, primary_key=True),
-        sa.Column("task_name", sa.String, primary_key=True),
+        sa.Column("id", sa.INTEGER, autoincrement=True),
```

Comment:
is this needed? since we'll have filing_period, and lei to relate to filing; and task_name to relate to the task table